test-colormap.cpp

```diff
@@ -0,0 +1,39 @@
+#include <assert.h>
+#include <cstring>
+#include <iostream>
+#include "./misaligned.h"
+#include "./tshirts.h"
+#include "./weatherreport.h"
+#include "./test-colormap.h"
+
+void testPrintColorMap() {
+    std::string result = printColorMap();
+    std::string expected =

```

Comment:
The expected needs to be close to what the consumers of the code would expect. They would expect according to the standard- pair numbers go from 1 to 25, not 0 to 24.
Also, they would expect the column separators to be aligned.
Make the test in such a way, that a "pass" will give confidence that it will be acceptable to the user